src/main/resources/com/rackspace/jenkins_nodepool/NodePoolSlave/configure-entries.jelly

```diff
@@ -0,0 +1,41 @@
+<!--
+  Config page
+-->
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout"
+         xmlns:t="/lib/hudson" xmlns:f="/lib/form">
+
+    NodePool agents/slaves are ephemeral and last for the duration of a single build.
+    <p/>
+
+    <f:entry title="${%Hold Node?}" field="held">
```

Comment:
This should be read-only after the build is complete, as it won't do anything. A user may try unchecking `Hold Node` in order to release it, but that won't work, they'll need to delete the computer instead (which will in turn release the NodePoolNode). 

```
<j:choose>
   <j:when test="${build_in_progress}"> <!-- checkbox here --></j:when>
   <j:otherwise>Node is held, please delete when no longer required</j:otherwise>
</j:choose>
```

The otherwise can always say the node is held, as it won't exist long enough for that statement to become inaccurate if it isn't  held..

Also I'm slightly surprised that this Jelly relates to the slave not the computer, is that because Computers aren't serializable so can't have config data bound to them?